server/src/modules/actions/emails.actions.ts

```diff
@@ -27,6 +27,9 @@ function addEmail(
           sendDates: [new Date()],
         },
       },
+      $set: {
+        updated_at: new Date(),
+      },
```

Comment:
Ici `upsert=true`, il faut prevoir le cas de create

```suggestion
      $set: {
        updated_at: now,
      },
      $setOnInsert: {
          created_at: now
      }
```